hack/boilerplate/boilerplate.Dockerfile.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 The cert-manager Authors.
+# Copyright YEAR The cert-manager Authors.
```

Comment:
Thank you for doing this! Could you please split out the boilerplate changes into a separate PR to have separation of concerns?